src/main/java/org/apache/commons/lang3/reflect/MemberUtils.java

```diff
@@ -260,12 +266,13 @@ private static boolean isMatchingExecutable(final Executable method, final Class
         }
 
         if (method.isVarArgs()) {
-            int i;
+            @NonNegative int i; // annotating as @NonNegative or else it is detected as @LowerBoundBottom by the checker
```

Comment:
Is there an issue you can link to here where you reported this? IIRC, it might have been https://github.com/typetools/checker-framework/issues/2505, which was fixed two weeks ago - which would mean we should be able to remove both this annotation and the comment.

Reply:
The checker shows lowerbound:array.access.unsafe.low when I remove the annotation.

Reply:
Can you write a test case and report a bug, then?